module/users/helper.js

```diff
@@ -1047,6 +1047,113 @@ module.exports = class UserHelper {
         })
     }
 
+        /**
+      * get usersObservationDocuments documents started by user.
+      * @method
+      * @name usersObservation
+      * @param  {String} userId - userId of user.
+      * @param  {String} solutionId - solution Id.
+      * @returns {result} - all the usersObservation in particular solution which user has submitted. 
+     */
+
+        static usersObservation({
+            stats,
+            userId
+        }){
+            return new Promise(async (resolve, reject) => {
+                try {
+
+                    let fields = [
+                        'name',
+                        'isAPrivateProgram',
+                        'entities',
+                        'status',
+                        'description',
+        
+                    ] 
+
+                    if(stats == 'true')
+                    {
+
+                        let countObservations = await observationsHelper.countDocuments({createdBy:userId})
+                        
+                        resolve({
+                            count:countObservations
+                        })
+                    }
+                    else{
+                        let observationDocument =
+                        await observationsHelper.observationDocuments(
```

Comment:
did you check if there is any aggregator function is there or not for this query? @borkarsaish65

Reply:
I have used aggregate function now